src/lib/components/buttonv2/Buttonv2.component.js

```diff
@@ -0,0 +1,231 @@
+//@flow
+import React from 'react';
+import type { Node } from 'react';
+import styled, { css } from 'styled-components';
+import Tooltip from '../tooltip/Tooltip.component.js';
+import type { Props as TooltipProps } from '../tooltip/Tooltip.component.js';
+import { getTheme } from '../../utils';
+import * as defaultTheme from '../../style/theme';
+import { spacing } from '../../style/theme.js';
+
+export type Props = {
+  variant?: 'primary' | 'secondary' | 'danger' | 'outline',
```

Comment:
Variant is mandatory in your implementation

Reply:
Good catch! 
After talking with @Cuervino, we realized we still need the `ghost button`, meaning the variant should be optional. So I removed the warning in 8e4f8d35